src/gpu/opengl/mod.rs

```diff
@@ -138,8 +139,18 @@ impl Renderer {
         // compile-time thing.
         let vs_src = include_str!("vertex.glsl");
         let fs_src = include_str!("fragment.glsl");
+        let prog_input = ProgramCreationInput::SourceCode {
+            vertex_shader: &vs_src,
+            tessellation_control_shader: None,
+            tessellation_evaluation_shader: None,
+            geometry_shader: None,
+            fragment_shader: &fs_src,
+            transform_feedback_varyings: None,
+            outputs_srgb: true,
```

Comment:
Could you add a comment here explaining that we set `outputs_srgb` to true because we don't want gamma correction and instead plan on doing color correction in the shader later on? Because technically this looks incorrect, we are really not outputting sRGB...